packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## NEXT
 
+## 2.4.13
 * Updates minimum supported SDK version to Flutter 3.13/Dart 3.1.
+* Call `onDestroy` instead of `initialize` in onDetachedFromEngine.
```

Comment:
Calls

(Please see the style guide linked from the PR checklist.)

Reply:
done.